spec/requests/graphql/query/digital_object_spec.rb

```diff
@@ -100,36 +100,32 @@
       sign_in_project_contributor to: :read_objects, project: authorized_project
       graphql query(authorized_object.uid)
     end
+    let(:expected_location) { Rails.root.join('spec', 'fixtures', 'files', 'test.txt').to_s }
 
     it "returns the expected resources response" do
       expect(response.body).to be_json_eql(%(
         [
           {
             "id": "master",
-            "checksum": "SHA256:e1266b81a70083fa5e3bf456239a1160fc6ebc179cdd71e458a9dd4bc7cc21f6",
             "displayLabel": "Master",
-            "fileSize": 23,
-            "location": "managed-disk://#{Rails.root.join('spec', 'fixtures', 'files', 'test.txt')}",
-            "mediaType": "text/plain",
-            "originalFilename": "test.txt"
+            "resource": {
+              "checksum": "SHA256:e1266b81a70083fa5e3bf456239a1160fc6ebc179cdd71e458a9dd4bc7cc21f6",
```

Comment:
It should, yeah.  This is a case of fixture data that needs to be updated.  I'll do that.  Thanks!

As a side note, I'm still trying to decide whether or not we want "urn:" in our Hyacinth-stored checksums, or if that's just a thing we want to include in Fedora data.  Do you or @barmintor have any preference about that?

Reply:
the thing I'm anxious about changing it from the current practice is pushing unnecessary updates to Fedora if the data appears to have changed when it didn't.

Reply:
To clarify, what I mean is that we're planning on writing the checksum to two places:
1) To the Hyacinth object's metadata json (i.e. the "primary" location)
and
2) In Fedora, upon publish, via: `@fedora_object.rels_int.add_relationship(content_ds, :has_message_digest, "urn:sha256:#{import_file_sha256_hexdigest}")`

Note: I copied the above code snippet from Hyacinth 2, just to be clear about the current form in Fedora.

So Fedora currently uses "urn:sha256:abcdefg" and I'm not proposing that we change that.  It makes sense to use a urn in that triple.  What I'm asking about is how we want to store the checksum on the Hyacinth side, in the metadata json for a resource:

```
{
  "resources" => {
    "master" => {
      "location" : "...",
      "checksum" : "?"
    }
  }
}
```
Do either of you have any preferences about storing the Hyacinth-side checksum string as "sha256:abcdefg" vs "urn:sha256:abcdefg"?  Whatever we decide, that string is returned by `digital_object.resources['master'].checksum` and if we don't reformat it, that's also the checksum string that users would see in the UI or CSV import/export.

I've been thinking it over and I think it might be easier NOT to include the "urn:" prefix in the internal Hyacinth format, both so we don't have to transform it when we display it or put it in a CSV and also so that people don't have to prefix the checksums in an input spreadsheet with "urn:".  Then, when exporting to Fedora, we'd prefix with "urn:" in that one and only place -- which is actually what the Fedora code originally assumed, before I modified it in this PR.

I just wanted to run all of this by both of you to make sure it's not a mistake to do what I'm proposing.  The previous Fedora code was implemented before this import code was finished, and based on the Hyacinth 2 method of doing things, and it may still be what we want to do.  Easy to revert -- just a couple of lines of code.

I recently made ticket HYACINTH-556 to make sure we handled Ben's earlier comment, but we can close it if we end up resolving things here.

Reply:
I don't have a strong opinion about this, but I also don't think I know enough about the use of URNs.

Reply:
Update/Recap: We spoke about this in a separate channel and we're going to use "sha256:abcdefg" for Hyacinth internal storage and "urn:sha256:abcdefg" for Fedora uri storage.